evergreen.yml

```diff
@@ -113,6 +117,19 @@ functions:
             --notary-url http://notary-service.build.10gen.cc:5000 \
             mongodb_jdbc.taco
 
+  "update plugin-version":
```

Comment:
I have a similar change in PR 11, I will merge the change based on which is checked in first.

Reply:
👍 So that it will update both manifest.xml and connectionProperties.js, correct?

Reply:
Yes, that's correct.